scripts/clang-format.py

```diff
@@ -14,7 +14,7 @@
     'lib',
     'master',
 ]
-husky_root = os.getenv('HUSKY_ROOT', '.')
+husky_root = '.';
```

Comment:
Because on master $HUSKY_ROOT in ENV does not apply for every user. User has to set `HUSKY_ROOT` to `.` anyway. If they are already instructed to `cd husky`, I think this would be better.